recipes/pulseaudio/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+import shutil
+
+
+class LibnameConan(ConanFile):
+    name = "pulseaudio"
+    description = "PulseAudio is a sound system for POSIX OSes, meaning that it is a proxy for sound applications."
+    topics = ("conan", "pulseaudio", "sound")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://pulseaudio.org/"
+    license = "LGPL-2.1"
+
+    generators = "pkg_config"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_alsa": [True, False],
+        "with_glib": [True, False],
+        "with_fftw": [True, False],
+        "with_x11": [True, False],
+        "with_openssl": [True, False],
+        # FIXME: enable when #2147 is merged
+        # "with_dbus": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_alsa": True,
+        "with_glib": False,
+        "with_fftw": True,
+        "with_x11": True,
+        "with_openssl": True,
+        # FIXME: enable when #2147 is merged
+        # "with_dbus": False,
+    }
+
+    build_requires = "gettext/0.20.1", "libtool/2.4.6"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _autotools = None
+    
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("pulseaudio supports only linux currently")
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+
+    def requirements(self):
+        self.requires("libsndfile/1.0.30")
```

Comment:
Is libsndfile required?
https://cgit.freedesktop.org/pulseaudio/pulseaudio/tree/configure.ac#n680

I'm asking because I am looking to package an application using audio with as little dependencies as possible.

Reply:
it looks like the line you copied would make the execution fail if libsndfile was not to be found (according to https://autotools.io/pkgconfig/pkg_check_modules.html), so I'm inclined to answer yes to your question

Reply:
Thanks for checking